src/components/SEP/MeetingComponents/SEPMeetingInstrumentsTable.tsx

```diff
@@ -31,8 +31,8 @@ const instrumentTableColumns = [
   },
   {
     title: 'Submitted',
-    render: (rowData: InstrumentWithAvailabilityTime) =>
-      rowData.submitted ? 'Yes' : 'No',
+    field: 'submitted',
+    lookup: { true: 'Yes', false: 'No' },
```

Comment:
it could be yes I will double check this but the thing is that the changes come from here: https://github.com/UserOfficeProject/user-office-frontend/pull/865. So that PR is older and should be reviewed first.